SPECS/perl/perl.spec

```diff
@@ -396,13 +453,24 @@ directories).
 Summary:        Header files for use in perl development
 # l1_char_class_tab.h is generated from lib/unicore sources:    UCD
 License:        (GPL+ or Artistic) and UCD
+%if %{with perl_enables_systemtap}
+Requires:       systemtap-sdt-devel
+%endif
 Requires:       perl(ExtUtils::ParseXS)
 Requires:       %perl_compat
 # Match library and header files when downgrading releases
 Requires:       perl-libs%{?_isa} = %{perl_epoch}:%{perl_version}-%{release}
 Recommends:     perl-doc = %{perl_version}-%{release}
 # Devel::PPPort for h2xs script
 Requires:       perl(Devel::PPPort)
+# Compiler and linker options stored into perl and used when building XS
+# modules refer to hardering profiles like
+# /usr/lib/rpm/redhat/redhat-hardened-cc1 that are delivered by
+# redhat-rpm-config. Bug #1557667.
+Requires:       redhat-rpm-config
```

Comment:
We don't have `redhat-rpm-config`. In our case we're using the `default-hardened-cc1` file provided by `mariner-rpm-macros`:

```suggestion
# Compiler and linker options stored into perl and used when building XS
# modules refer to hardering profiles like
# /usr/lib/rpm/redhat/default-hardened-cc1 that are delivered by
# mariner-rpm-macros.
Requires:       mariner-rpm-macros
```

We should probably check what that "Bug #1557667" is about, to make sure Perl will still work for us.

Reply:
Good catch, will update this to use our mariner-rpm-macros package. I believe this bug would apply to us as well

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1557667